src/pages/projectSettings/index.tsx

```diff
@@ -95,12 +95,16 @@ const ProjectSettings: React.FC = () => {
     identifier,
     currentTab: tab,
   };
-
   const project =
     projectType === ProjectType.Repo
       ? repoData?.repoSettings
       : projectData?.projectSettings;
 
+  const hasLoaded =
+    (projectData || isRepo) &&
+    (repoData || projectType === ProjectType.Project) &&
+    project;
```

Comment:
Can this be rewritten as 
```suggestion
    const hasLoaded = !projectLoading && !repoLoading && project;
```
where `repoLoading` was added to the repo query? I ran a test patch [here](https://spruce.mongodb.com/version/651b0aff3627e0ac130b54ad/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

Reply:
If not, I think I would at least prefer
```suggestion
  const hasLoaded =
    (projectData ||  projectType === ProjectType.Repo) &&
    (repoData || projectType === ProjectType.Project) &&
    project;
```